source/editableText.py

```diff
@@ -40,21 +41,24 @@ class EditableText(TextContainerObject, ScriptableObject):
 	@type hasContentChangedSinceLastSelection: bool
 	"""
 
-	#: Whether to fire caretMovementFailed events when the caret doesn't move in response to a caret movement key.
-	shouldFireCaretMovementFailedEvents = False
-
-	#: Whether or not to announce text found before the caret on a new line (e.g. auto numbering)
-	announceNewLineText = True
-	#: When announcing new line text: should the entire line be announced, or just text after the caret?
-	announceEntireNewLine = False
-
-	#: The minimum amount of time that should elapse before checking if the word under the caret has changed
-	_hasCaretMoved_minWordTimeoutSec = 0.03
-
-	#: The maximum amount of time that may elapse before we no longer rely on caret events to detect movement.
-	_useEvents_maxTimeoutSec = 0.06
-
-	_caretMovementTimeoutMultiplier = 1
+	shouldFireCaretMovementFailedEvents: bool = False
+	"""Whether to fire caretMovementFailed events when the caret doesn't move in response to a caret movement key."""
+	announceNewLineText: bool = True
+	"""Whether or not to announce text found before the caret on a new line (e.g. auto numbering)"""
+	announceEntireNewLine: bool = False
+	"""When announcing new line text: should the entire line be announced, or just text after the caret?"""
+	_hasCaretMoved_minWordTimeoutSec: float = 0.03
+	"""The minimum amount of time that should elapse before checking if the word under the caret has changed"""
+	_useEvents_maxTimeoutSec: float = 0.06
+	"""The maximum amount of time that may elapse before we no longer rely on caret events to detect movement."""
+	_caretMovementTimeoutMultiplier: Number = 1
```

Comment:
should this be a `Real` not a `Number`? I think `Number` includes complex numbers

Reply:
Sounds reasonable to me.